services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2542,6 +2549,19 @@ public void updateStore(String clusterName, String storeName, UpdateStoreQueryPa
         addWriteComputeSchemaForStore(clusterName, storeName, true);
       }
 
+      if (!veniceHelixAdmin.isHybrid(currStore.getHybridStoreConfig())
```

Comment:
Question:
If we setup the partition count for a new hybrid store, will the first version push affect the partition count of RT any more?
If not, does it mean all the new hybrid stores will always use the same default partition count?

Reply:
1. No. Once the store-level partition count is set, the first version will use store-level partition count.
2. No. If store owners/operators specify partition count when enabling hybrid/inc push, the store won't use default hybrid partition count.

Reply:
For 2), yes, and this is tricky to allow the customer to specify the partition count at conversion time.
But that is fine. Once hybrid store repartitioning feature is available, the hybrid store partition count will become adjustable.